apps/client/src/app/pages/Home.tsx

```
@@ -0,0 +1,13 @@
+import { Map, Routing } from '@/domains';
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)